webhookHandler.go

```diff
@@ -0,0 +1,107 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/go-kit/kit/log"
+	"github.com/xmidt-org/argus/webhookclient"
+	"github.com/xmidt-org/webpa-common/logging"
+	"github.com/xmidt-org/webpa-common/webhook"
+	"io/ioutil"
+	"net/http"
+	"time"
+)
+
+type Registry struct {
+	hookStore *webhookclient.Client
+	config    RegistryConfig
+}
+
+type RegistryConfig struct {
+	Logger      log.Logger
+	Listener    webhookclient.ListenerFunc
+	ArgusConfig webhookclient.ClientConfig
+}
+
+func NewRegistry(config RegistryConfig, listener webhookclient.Listener) (*Registry, error) {
+	argus, err := webhookclient.CreateClient(config.ArgusConfig, webhookclient.WithLogger(config.Logger))
```

Comment:
where are we registering the config.Listener (caduceusSenderWrapper.Update in this case)?